### ISSUE TITLE
Command failed to execute properly

### ISSUE THREAD

**Issue by wmliang** [hugsy/gef#203]
it seems some common task failed
the issue started from 1722cc2fecc58c4e391a0395e1f48981c6bcb8a5
```
gef➤  heap arenas
[!] Command 'heap arenas' failed to execute properly, reason: 'NoneType' object has no attribute 'address'
gef➤  heap bins fast
───────────────────────────────────────────────────────────────────────────────────────[ Fastbins for arena 0x7ffff7dd1b20 ]───────────────────────────────────────────────────────────────────────────────────────
[!] Command 'heap bins fast' failed to execute properly, reason: 'NoneType' object has no attribute 'address'
Fastbins[idx=0, size=0x10] 
```

**Comment by hugsy** [comment]
Thanks for reporting. I confirm the bug, it is a regression due to `dereference()` now forcing lazy fetch. I will fix it to `master` quickly (and add regression tests on `heap` commands).

@wmliang if you need a quick patch in the mean time, apply this:
```patch
diff --git a/gef.py b/gef.py
index c451c68..523f948 100644
--- a/gef.py
+++ b/gef.py
@@ -616,7 +616,8 @@ class GlibcArena:
         return self.__addr

     def dereference_as_long(self, addr):
-        return long(dereference(addr).address)
+        deref = dereference(addr)
+        return 0 if deref is None else long(dereference(addr).address)

     def fastbin(self, i):
         addr = self.dereference_as_long(self.fastbinsY[i])
```

**Comment by hugsy** [comment]
@wmliang It should be fixed in bb77c36. Feel free to test it and close the ticket if it works out for you too.

Cheers

**Comment by wmliang** [comment]
bug fixed, thanks 👍
